### DISCUSSION TITLE
Welcome to Seedvault Discussions!

### DISCUSSION THREAD

**Discussion by chirayudesai** [seedvault-app/seedvault#178] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chirayudesai** [comment]
Starting off myself,

Hello,

I'm Chirayu Desai, I work on CalyxOS at The Calyx Institute.
For Seedvault, I do a lot of release management, testing, and most importantly, code review!

**Comment by stevesoltys** [comment]
Hey! I'm Steve. I wrote the original implementation of Seedvault, before the folks at Calyx were kind enough to start contributing :smile:

More recently I have been doing project management and code reviews. I am happy to help with any questions you may have about the project.

**Comment by beerisgood** [comment]
Hello, I'm beerisgood and I'm just a normal user which use (and love) Seedvault in GrapheneOS :D

**Comment by noxamox23** [comment]
Hi, I recently installed Lineageos on my Pixel 3 and am quite happy with it.
I now wanted to see what seedvault is about but was not yet ready to really use it, so I not bother about writing down the recovery code I was shown at the beginning. I expected that I would be able to just start the setup assistant again anytime.
Now there seems to be no way to get back to the page with the recovery code. There also does not seem to be any possibility to set a new recovery code. 
Just to clarify: I did not yet make any backup. 
Any info how to reset seedvault without having to set up the phone completely anew?

**Reply by beerisgood** [reply]
> Hi, I recently installed Lineageos on my Pixel 3 and am quite happy with it.
> I now wanted to see what seedvault is about but was not yet ready to really use it, so I not bother about writing down the recovery code I was shown at the beginning. I expected that I would be able to just start the setup assistant again anytime.
> Now there seems to be no way to get back to the page with the recovery code. There also does not seem to be any possibility to set a new recovery code.
> Just to clarify: I did not yet make any backup.
> Any info how to reset seedvault without having to set up the phone completely anew?

You can start SeedVault setup as often as you want.
It is also ready to use, don't know why you say that. Sounds like faulty LineageOS.

**Reply by noxamox23** [reply]
Hi beerisgood, thanks for trying to help.
I'm on LineageOS 17.1 build 30 March 2021 for Pixel 3 (blueline).
When I go to Settings->System->Advanced->Backup 
I get to a screen showing "Seedvault Backup" and "Change backup provider".
If I tap on "Seedvault Backup" I get to the Settings for Seedvault but there is no option to restart the setup.
If I tap on "Change backup provider" I get to a screen with Seedvault as the only choice - that throws me back to the previous screen showing "Seedvault Backup" and "Change backup provider".
So, I do not see any way to "start SeedVault setup".
NB: I do not have any gapps installed.

**Reply by chirayudesai** [reply]
In a future update you will be able to set a new code from within Seedvault.

Till then, you can go to Settings -> Apps -> Menu -> Show system -> Seedvault -> Storage -> Clear data to start over.

**Reply by noxamox23** [reply]
chirayudesai U rock!
